compose.sim.webots.yaml

```diff
@@ -0,0 +1,64 @@
+services:
+
+  rviz:
+    image: husarion/rviz2:humble-22-12-20
+    runtime: nvidia
+    container_name: rviz
+    network_mode: host
+    ipc: host
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+      - ./config/rosbot.rviz:/root/.rviz2/default.rviz
+    environment:
+      - RMW_IMPLEMENTATION
+      - DISPLAY=${DISPLAY:?err}
+      - NVIDIA_VISIBLE_DEVICES=all
+      - NVIDIA_DRIVER_CAPABILITIES=all
+
+  rosbot-xl:
+    image: husarion/webots:galactic
+    runtime: nvidia
+    network_mode: host
+    ipc: host
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+    environment:
+      - DDS_CONFIG
```

Comment:
I'm not sure if `DDS_CONFIG` is necessary (also in the `navigation` and `localization`).

Reply:
The separate `nav2` and `amcl` configurations were caused by a lack of `laser_filters`.
Here I added `laser_filters':
- https://github.com/husarion/webots-docker/commit/cf0d473b4e2cf7a4aa0786ad6ff3dd9b82307ecf
- https://github.com/husarion/webots_ros2/commit/8af5ce10dbb2b1ff8a087efc6f482c270f266e79